### ISSUE TITLE
Donate link not working /cant make donations

### ISSUE THREAD

**Issue by mkorthof** [chocolatey/chocolatey.org#540]
### What You Are Seeing?
Donate/contribute link not working at 
- https://chocolatey.org/pricing
- https://github.com/chocolatey/choco (top of README.md)

### What is Expected?
Being able to make a donation

### How Did You Get This To Happen? (Steps to Reproduce)
1) Click `receives: invalid`
2) Link refers to https://www.gratipay.com/Chocolatey but www.gratipay.com does not resolve

Seems gratipay shutdown end of 2017 (https://gratipay.news/the-end-cbfba8f50981)

**Comment by ferventcoder** [comment]
Yes, apologies. We knew they were shutting down but have not updated any links yet. It will likely be at least a couple of weeks before we have an alternative selected

**Comment by bcurran3** [comment]
Why not switch to Patreon like most media creators? Or PayPal like the majority of developers?

Seems every month (6 so far) you're losing out on possible donations/income/beer money/etc.

**Comment by ferventcoder** [comment]
We are looking at something - being a for profit company behind Chocolatey nowadays always makes me feel a bit weird to accept donations. However I think if we find a better way to transfer those donations towards open source efforts (or tools!!) then I think that might be acceptable. As long as we find a way to transparently do so.

**Comment by bcurran3** [comment]
Gary and I were discussing this the other day (some of your thoughts via proxy).

Two Ideas:
1 - Change Chocolatey donation links to a list of your preferred/suggested/causeworthy OSS donation links.
2 - Accept donations for Chocolatey but pass it on to the volunteers. Monies could be parceled out/awarded to code contributors (similar to a bug bounty), moderators, packagers, etc. These passed along donations could be called Chocolatey Gratitudes(tm) and don't even need to be in the form of actual money. Money collected could be used/converted to pay cost of goods to send out t-shirts and other sundry "thank you's" via the Chocolatey shop. (If the people over at cafe press ever saw this, I'm sure they'd upvote!) 

Being a packager, of course, my vote would go for option 2. I think it's great to support other worthy OSS projects, but I think donations would be better served in-house.
